content/pt-br/docs/concepts/storage/volumes.md

```diff
@@ -637,9 +662,10 @@ Como operador do cluster Kubernetes que administra o armazenamento, aqui estão
 
 Um volume `secret` é usado para passar informações sensíveis, tais como senhas, para Pods. Você pode armazenar segredos na API Kubernetes e montá-los como arquivos para serem usados por pods sem necessidade de vinculação direta ao Kubernetes. Volumes `secret` são mantidos pelo tmpfs (um sistema de arquivos com baseado em memória RAM) para que nunca sejam gravados em armazenamento não volátil.
 
-{{< note >}}Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo. {{< /note >}}
-
-{{< note >}} Um contêiner que utiliza um Secret como ponto de montagem para a propriedade [`subPath`](#using-subpath) não receberá atualizações deste Secret. {{< /note >}}
+{{< note >}}
+* Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo. {{< /note >}}
```

Comment:
```suggestion
* Você deve criar um Secret na API Kubernetes antes de poder utilizá-lo.
```

It seems an extra closure for the `{{< note >}}` shortcode was left here, and that's causing the build to fail.